aws/extensions/computetask_awscli/extension.ftl

```diff
@@ -26,6 +26,9 @@
 
     [#local content = {}]
     [#switch operatingSystem.Family ]
+        [#case "windows" ]
+        [#break]
+
```

Comment:
Are we not installing the aws cli? or is it already bundled?

Reply:
Doco says AWS CLI is already bundled